src/components/informationCard/InformationCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './InformationCard.css';
+import {ArrowRight} from "@phosphor-icons/react";
+
+const InformationCard = ({infoCardTitle, infoCardContent, experienceNeeded, onClick}) => {
+    return (
+        <div className="infoCard">
+            <h3 className="info-card-title">{infoCardTitle}</h3>
+            <p className="info-card-content">{infoCardContent}</p>
+            {experienceNeeded && <p className="info-card-experience-tag"><em>Enige IT-ervaring vereist</em></p>}
+            <button
```

Comment:
Omdat deze button geen tekst bevat, is het netjes om er een [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) aan toe te voegen voor accessability. MDN zegt: _"Sometimes, the default accessible name of an element is missing or the accessible name does not accurately describe the contents of the element and there is no content visible in the DOM that can be associated with the object to give it meaning. A common example of such an element is a button containing an SVG icon without any text."_

Reply:
@MorikuMarieke deze nog